modules/ROOT/pages/errors/all-errors.adoc

```diff
@@ -1,29 +1,109 @@
 :description: The Neo4j error codes for Neo4j version {neo4j-version}.
 
-
 [[neo4j-errors]]
-= List of all server error codes
+= List of all Neo4j 5 server error codes
+
+Error codes are Neo4j status codes returned by the server when the execution of a query fails.
+They always have the severity level `ERROR`.
+Errors are grouped based on the type of the error code:
+
+Client errors::
+    These errors are caused by the client and are usually related to the request itself.
+    Client errors have the prefix `Neo.ClientError`.
+
+Transient errors::
+    These errors are detected by the server and are usually related to some kind of database unavailability, such as limits reached, out-of-memory, timeouts, etc.
+    The error can be temporary and could therefore succeed if retrying the request.
+    Transient errors have the prefix `Neo.TransientError`.
+
+Database errors::
+    These errors are caused by the database and are usually related to the database state.
+    Database errors have the prefix `Neo.DatabaseError`.
 
 This page contains lists of all Neo4j errors, grouped by type.
 
+[[neo4j-client-errors]]
 == Client errors
 
 This is a complete list of all client errors Neo4j may return, and what they mean.
 
+[[Neo.ClientError.ChangeDataCapture.InvalidIdentifier]]
+===  `Neo.ClientError.ChangeDataCapture.InvalidIdentifier`
+
+This error occurs when the given change identifier is invalid.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|Description
+a| Invalid change identifier.
+|GQLSTATUS code(s)
```

Comment:
I think we should clarify that this is a cause GQLSTATUS code in some way. In the next table you have the top level code in the same place and that distinction would be important to know for someone adapting their code to the new system

Reply:
Hmm, this shouldn't be the cause code. This is the top-level code.

Reply:
I used the spreadsheet https://docs.google.com/spreadsheets/d/1VNF5uO3LfoUGYQkewAA7H3IoNmXNmhSZlUOf3EIvfUw/edit?gid=1522141749#gid=1522141749